### PR TITLE
Handle correction layer offsets

### PR DESCRIPTION
**Pull request by sssangha**
Leverage @mgovorcin sequential stitching implementation for SET and tropo correction layers.

### CODE REVIEW

**Comment by dbekaert** [comment]
per discussion; i beleive this is covering up an underlying problem in the SET code.

File: tools/ARIAtools/tsSetup.py [hunk 1]
```diff
@@ -457,7 +461,7 @@ def main(inps=None):
         'mask': inps.mask,
         'outDir': inps.workdir,
         'outputFormat': inps.outputFormat,
-        'stitchMethodType': 'overlap',
+        'stitchMethodType': inps.stitchMethodType,
```
Comment (dbekaert) [inline tools/ARIAtools/tsSetup.py:464]: good catch

**Review by dbekaert — APPROVED** [review]
LGTM So looks like we are not supporting 3.8 anymore with the recent changes.

File: tools/ARIAtools/tsSetup.py (hunk 1 above)
Comment (sssangha) [reply tools/ARIAtools/tsSetup.py:464]: Yeah, but tbf I think that was intentional before... I.e. for ariaTSsetup the whole `stitchMethodType` argument had been commented out to enforce overlap stitching as 2stage unwrapping was only other choice and experimental.

**Comment by sssangha** [comment]
> LGTM So looks like we are not supporting 3.8 anymore with the recent changes.

Yup, which reminds me to update the readme! In summary, the new version of rioxarray utilizes tuples in a way that is no longer compatible with [python3.8](https://docs.python.org/3.8/library/typing.html#type-aliases) vs python 3.9 and [above](https://docs.python.org/3.9/library/typing.html#type-aliases) as @cmarshak discussed with me..

Something to track in our other repos!

**Comment by sssangha** [comment]
@mgovorcin I cleaned up the sequential stitcher a little bit, namely making it inline with pep8 standard of a max of 79 characters per line + adjusting the variables in the metadata layer stitcher I introduced

**Review by mgovorcin — APPROVED** [review]

**Pull request merged**